nemo_skills/inference/server/model.py

```diff
@@ -272,7 +271,7 @@ def _send_request(
         random_seed: int,
         stop_phrases: list[str],
     ) -> str:
-        messages = prompt.build_structured(input_dict)
+        messages = self._parse_prompt(prompt)
```

Comment:
Let's remove build_structured from prompt class as we don't need it anymore. We can always bring it back in the future if there is a reason to use it